.github/workflows/lint_code.yml

```diff
@@ -46,4 +46,6 @@ jobs:
         id: checkov
         uses: bridgecrewio/checkov-action@master
         with:
-          args: --quiet --framework terraform
+          quite: true
+          framework: terraform
+          skip_check: CKV_AWS_51,CKV_AWS_136
```

Comment:
It will be changed in the next PR that setting up ECR 🙏🏻. ([ref](https://github.com/sanG-github/nimble-devops-ic-infrastructure/blob/3d6b3e57ebd181328d3b7d98c4b9f795cdd0f761/.github/workflows/lint_code.yml#L48-L50), [ref](https://github.com/sanG-github/nimble-devops-ic-infrastructure/blob/3d6b3e57ebd181328d3b7d98c4b9f795cdd0f761/modules/ecr/main.tf#L3-L5))

Reply:
Updated it in 89d4708 ✅ ✨